source/guide/3.2/webui-blueprints-page.md

```diff
@@ -7,21 +7,38 @@ abstract: Blueprint Page Reference
 pageord: 110
 
 terminology_link: reference-terminology.html
+node_types_link: dsl-spec-node-types.html
+relationships_link: dsl-spec-relationships.html
 ---
 {%summary%}This is a view of a single blueprint in Cloudify's User Interface{%endsummary%}.
 
 When clicking on the `Blueprints` tab and choosing a blueprint you will be able to choose one of the following:
 
 # Topology
-See the definition [here]({{page.terminology_link}}#topology).<br/>
+The [Topology]({{page.terminology_link}}#topology) is an application’s graph of nodes and their relationships, which describes the lifecycle events or other operations that each node and relationship exposes for use in workflows.<br>
+Each of the blueprint nodes is displayed as a square container, which can contain another nodes. Each of the nodes has a title describing its name, and an icon to indicate the [node type]({{page.node_types_link}}).<br>
```

Comment:
blueprint nodes -> blueprint's nodes
another nodes -> other nodes...

I see some of these around, which I already commented on. please fix.